LOCAL_README.md

```diff
@@ -0,0 +1,74 @@
+## Prerequisites
```

Comment:
Codacy found an issue: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/lgruelas/finance/pullRequest?prid=5716470)